### PR TITLE
Add My Profile : prasundas99.json

### PR DESCRIPTION
**Pull request by Prasundas99**
This PR Proposes Changes to `/public/data`

 ## Changes Proposed to
 Added kiritocode1.json to the directory `/public/data` .
 
 ## Check List
 * [x]  Compiles to the Code Style of the project ✍️
 * [x]  New and Existing test passed
 * [x]  Does Not Contain Copyright Exclusive Content of any type .
 * [x]  Describes what  changes i made .
 
 ## Screenshots
image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1952"></a>

### CODE REVIEW

File: public/data/PrasunDas99.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Prasun Das",
+  "type": "personal",
+  "bio": "SDE @Quantorix Ex-IEMLabs Ex-Aberrantz| Trainer | Web Developer| Microservices | Open Source Contributor | Blockchain | Innovation Enthusiastic",
+  "avatar": "https://github.com/prasundas99.png",
+  "links": [
+    {
+      "name": "Protfolio",
```
Comment (Panquesito7) [inline public/data/PrasunDas99.json:8]: Spelling error.
```suggestion
      "name": "Portfolio",
```

File: public/data/PrasunDas99.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Prasun Das",
+  "type": "personal",
+  "bio": "SDE @Quantorix Ex-IEMLabs Ex-Aberrantz| Trainer | Web Developer| Microservices | Open Source Contributor | Blockchain | Innovation Enthusiastic",
```
Comment (Panquesito7) [inline public/data/PrasunDas99.json:4]: I recommend adding extra spaces here. What do you think? 🙂
```suggestion
  "bio": "SDE @Quantorix Ex-IEMLabs Ex-Aberrantz | Trainer | Web Developer | Microservices | Open Source Contributor | Blockchain | Innovation Enthusiastic",
```

File: public/data/PrasunDas99.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      "description": "Contributed to Twillo",
+      "url": "https://github.com/Prasundas99"
+    },
+    {
+      "title": "Started PepHub",
+      "date": "July 2022",
+      "icon": "book",
+      "color": "green",
+      "description": "Won top teacher award from GitHub.",
+      "url": "https://github.com/eddiejaoude"
```
Comment (Panquesito7) [inline public/data/PrasunDas99.json:43]: You might want to update this link.

Comment (Prasundas99) [reply public/data/PrasunDas99.json]: Didn't realized that. updated

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 👍

**Pull request merged**